File: JAVA/2108. Find First Palindromic String in the Array.java
class Solution {
    public String firstPalindrome(String[] w) {
        for(int i=0;i<w.length;i++){
            if(is(w[i])==true)
            return(w[i]);
        }
        return "";
    }
    private boolean is(String s){
        int i;
        for(i=0;i<s.length()/2;i++){
            if(s.charAt(i)!=s.charAt(s.length()-i-1))
            return false;
        }
        return true;
    }
}

File: C++/2108. Find First Palindromic String in the Array.cpp
class Solution {
public:
    bool is(string s){
        for(int i=0;i<s.size()/2;i++){
            if(s[i]!=s[s.size()-i-1])
            return false;
        }
        return true;
    }
    string firstPalindrome(vector<string>& w) {
        for(string s:w){
            if(is(s))
            return s;
        }
        return "";
    }
};